src/javaRosa.js

```diff
@@ -1388,6 +1388,9 @@ define([
                     insertOutputRef(_this, target, path, mug);
                     if (window.analytics) {
                         window.analytics.usage("Output Value", "Drag and Drop");
+                        if (path.contains('casedb')) {
+                            window.analytics.usage("Case Management", "Form Builder", "Drag and Drop");
+                        }
```

Comment:
The second part of my [previous comment](https://github.com/dimagi/Vellum/pull/625/files/21badeee3d5285ffb09b39f815bbcbed83197a51#r56562730) has not been addressed yet: the arguments for the `usage` function are `category, action, label, value`. If I'm not mistaken action is "Drag and Drop" on line 1390 and "Form Builder" on line 1392—is that intended?